datasource/api/src/main/java/org/opennms/oce/datasource/api/AlarmHandler.java

```diff
@@ -30,8 +30,13 @@
 
 public interface AlarmHandler {
 
-    void onAlarmCreatedOrUpdated(Alarm alarm);
+    default void onAlarmCreatedOrUpdated(Alarm alarm) {
+    }
 
-    void onAlarmCleared(Alarm alarm);
+    default void onAlarmCleared(Alarm alarm) {
+    }
+
+    default void onSituationAlarm(Incident incident) {
```

Comment:
Can we move this to a new `IncidentHandler` instead? I'd like to continue to keep the notions of alarms and situations  distinct in this project.